www/docs/content/recipes/digital-products.mdx

```diff
@@ -0,0 +1,1141 @@
+---
+addHowToData: true
+---
+
+import DocCardList from '@theme/DocCardList';
+import DocCard from '@theme/DocCard';
+import Icons from '@theme/Icon';
+
+# Digital Products
+
+This document guides you through the different documentation resources that will help you build digital products with Medusa.
+
+## Overview
+
+A business can sell digital or downloadable products. The products are stored privately using a storage service, such as S3, or other storage mechanism. When the customer buys this type of product, typically an email is sent to them where they can download the product.
+
+Medusa doesn’t support digital products by default. However, it provides the customization capabilities necessary to implement it based on your use case, restriction free. You can create new entities that allow you to represent a digital product and link them to existing product-related entities.
```

Comment:
Would change the first sentence to:

"While Medusa doesn't have a built-in concept of a digital product, it provides ..."

Reply:
Might also be worth addressing why we haven't included this by default - i.e. digital products can have many nuances that make standardization hard to do well, instead we offer building blocks that enable you to build the right experience easily.